nfc/include/nrf_nfc_errno.h

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#ifndef NRF_NFC_ERRNO_H__
+#define NRF_NFC_ERRNO_H__
+
+/**
+ * @defgroup nrf_nfc_errno Error codes for nRF NFC libraries.
```

Comment:
This doxygen group doesn't seem to be linked to any .rst file, so I think it doesn't appear in the rendered documentation (or I missed it?). Is this intentional? An entry in CHANGELOG.rst states the aligned error codes are documented in the API docs.

Reply:
Yes by the API I mean API in NFC header files. I documented all possible return values in functions doc